### ISSUE TITLE
Map/chart additional screenreader support

### ISSUE THREAD

**Issue by adamkendis** [hackforla/311-data#809]
# Description
The map and charts currently provide no useful information for screen reader users. A written representation of what is displayed on the map/charts should be provided for screen readers.

### 1. Map
 - User selects:
   - Last Month
   - Arleta
   - Bulky Items, Metal/Household Appliances

 - User tabs to Submit button and hits space.

 - User tabs to the map (this should be the first tab after Submit).

 - Screenreader reads something like:
   - "Map displaying **<TOTAL_COUNT_OF_REQUESTS>** **Bulky Item, Metal/Household Appliances** 311 requests for **last month** in **Arleta** neighborhood council. **Arleta** neighborhood council had **<COUNT_FOR_REQ_TYPE>** **Bulky Item** requests, **<COUNT_FOR_REQ_TYPE>** **Metal/Household Appliances** requests."

### 2. Charts
This will probably be trickier and result in much more text than the map aria-label. Similar implementation approach. 

  - Time-to-close
       - "**<TOTAL_COUNT>** **Bulky Items** requests. Minimum time to close was **<NUMBER_OF_DAYS>**  days. Maximum time to close was **<NUMBER_OF_DAYS>** days. 25% were closed in **<NUMBER_OF_DAYS>** days, 50% were closed in **<NUMBER_OF_DAYS>**  days, 75% were closed in **<NUMBER_OF_DAYS>**  days."
    - Same for other request types.

  - Frequency
    - Follow points along x-axis for each request type. Include the date and count for each point.

 - Total requests
    - Same as Frequency

-  Pie charts
    - Report percentage for each category in each pie chart.

## Action Items
Map:
 - [x] Implement function to generate a string summarizing the map data - see example above. It might be more efficient to generate this string on the back end at query time and return it to the client with the response.
 - [x] Add aria-label with string from action item 1 to main map element.
 - [x] Disable tabbing to individual markers/clusters on the map.

Charts:
 - [x] Make charts tabbable.
 - [x] Add aria-label to each chart per examples above.
